src/Components/CreateAccount/CreateAccount.js

```diff
@@ -1,9 +1,54 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './CreateAccount.scss';
+import banner from '../../images/banner.jpg'
+
 
 const CreateAccount = () => {
+  const [nameInput, handleName] = useState('');
```

Comment:
Could we not refactor this to just be one state object and utilize e.target.value to overwrite specific key value pairs in state instead of having multiple hooks?

Reply:
Good question, Chris. I started out with a dynamic hook and was getting an error that stated that the code was making a controlled form uncontrolled. I learned it was because I needed to break each out into its own hook.